src/components/Card/Card.tsx

```diff
@@ -0,0 +1,70 @@
+import { useEffect, useState } from "react";
+import React from 'react'
+import { CardStyled, CardHeader, CardFooter } from "./Card.styles";
+import { CardProps } from "./types";
+import {iconTypes} from "../Icon/collection";
+import colors from "../../styles/colors";
+import {Icon} from "../Icon";
+import {Tooltip} from "../Tooltip";
+
+const Card: React.FC<CardProps> = ({
+    id,
+    children,
+    description,
+    selected,
+    title,
+    tooltipText = "Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolores corporis vero in ipsum temporibus nesciunt!",
+}: CardProps) => {
+    const [ isSelected, setSelected ] = useState<boolean | undefined>(selected);
+    const [showCheckedIcon, toggleChecked] = useState<boolean | undefined>(false);
+
+    useEffect(() => {
+        toggleChecked(selected);
+    }, [selected]);
+
+    return (
+        <CardStyled
+            id={id}
+            data-testid={"card-test-id"}
+            onClick={() => {
+                setSelected(!isSelected);
+                toggleChecked(!showCheckedIcon);
+            }}
+            selected={isSelected}
+        >
+            <CardHeader data-testid={"header-test-id"}>
+                { showCheckedIcon &&
+                    <Icon
+                    data-testid={"check-test-id"}
+                    fill={colors.green}
+                    size={24}
+                    svg={iconTypes.checkmark}
+                    />
+                }
+                <Tooltip
+                    data-testid={"tooltip-test-id"}
+                    children={
+                        [
+                            <Icon
+                                key={"ttip-card"}
```

Comment:
was the console complaining about this needing a key? if its in a loop then it would need to be dynamic... but i dont think so in this case. hmmm i'm confused

also just FYI you need need to pass strings as JS
```key={"sting"} === key="string"```

Reply:
Yup i need to pass in a key. Otherwise tests cry around